doc/tutorials/gpu-passthru.rst

```diff
@@ -26,6 +26,8 @@ Consider the following dependencies and constraints:
 
 * When a device is assigned to a VM via GVT-d, no other VMs can use it.
 
+* For ADL-ASRock board, need disable BIOS setting "Above 4G Decoding"(under Advanced Menu -> SA Configuration) to enable GVT-d feature.
+
```

Comment:
@amyreye can you provide some wording here? I would recommend that we refer to that board as the `Alder Lake-based ASRock system`, or more simply the `ASRock system`

Reply:
@amyreye , please be free to close this PR and file new PR with your revised wording for this change. Thanks!

Reply:
Will merge this one and start a new PR to fix the board name as @gvancuts recommended.